Dockerfile-dev

```diff
@@ -0,0 +1,26 @@
+#
+# Dockerfile for an image with the currently checked out version of zipline installed. To build:
+#
+#    docker build -t quantopian/ziplinedev -f quantopian/ziplinedev .
```

Comment:
Almost there...
```bash
$ docker build -t quantopian/ziplinedev -f quantopian/ziplinedev .
Cannot locate Dockerfile: quantopian/ziplinedev
```
Looks like we're using the image name instead of the file name:
```bash
$ docker build -t quantopian/ziplinedev -f ./Dockerfile-dev .
Sending build context to Docker daemon 
Step 0 : FROM python:2.7
...
```

Mind just fixing that up, and we'll get this merged?  Thanks so much!